ui/lib/sync/addon/components/secrets/sync-activation-modal.ts

```diff
@@ -38,17 +37,16 @@ export default class SyncActivationModal extends Component<Args> {
     // sync activation is managed by the root/administrative namespace so child namespaces are not sent.
     // for non-managed clusters the root namespace path is technically an empty string so we pass null
```

Comment:
sounds like before we we're passing `null` and now we're passing undefined. I'm assuming this is totally fine, but wanted to gut check